core/shared/src/test/scala/scodec/bits/ByteVectorTest.scala

```diff
@@ -33,7 +33,7 @@ package scodec.bits
 import java.io.ByteArrayOutputStream
 import java.nio.ByteBuffer
 import java.util.{Arrays, UUID}
-
+import scala.io.Source
```

Comment:
@jendakol this seems to cause the issues with Scala.js
https://stackoverflow.com/a/40167262

But maybe there's a way to write the test that won't require `scala.io.Source`.